en/configuring.texy

```diff
@@ -338,3 +337,20 @@ For example, this is definition of service named `database` which is `PDO` insta
 services:
 	database: PDO('mysql:host=127.0.0.1;dbname=test', root, password)
 \--
+
+
+
+More extensions
+===============
+
+Through the extension, the configuration file can be supplemented by additional sections. Through these sections which we can configure components or modify the DI container.
+
```

Comment:
Asi bych to napsal takto, ale EN není má silná stránka *Using extensions, the configuration file can be enhanced by additional sections through which we can configure additional components or modify the DI container.*